### PR TITLE
Change CDN for Swagger UI

### LINKED ISSUE #6: Swagger UI's stylesheets aren't loaded
**Describe the bug**
When accessing the `/docs/swagger` route, a blank page is loaded. It seems to be due to the defined stylesheets no longer existing.

**Screenshots**
Swagger UI blank page

**To Reproduce**
Load `sanic-ext` into your Sanic app and access the Swagger UI route.

**Expected behavior**
The Swagger UI is displayed.

**Environment (please complete the following information):**
 - OS: Windows 10 Home 21H1
 - Browser Mozilla Firefox
 - Version: 94.0.1 64-bit

### PR DESCRIPTION
**Pull request by ahopkins**
Closes #6 

It seems that the CDN we were using no longer hosts the Swagger UI assets. Furthermore, the Cloudflare CDN is version only (no `latest`). Therefore, this PR does two things:

1. Changes the CDN to CF
2. Introduces a config value to change the swagger version

### CODE REVIEW

**Review by prryplatypus — DISMISSED** [review]
Looks good. Thanks!

File: sanic_ext/extensions/openapi/ui/swagger.html [hunk 1]
```diff
@@ -15,8 +16,9 @@
 
     <body>
         <div id="openapi">
-            <script src="//pagecdn.io/lib/swagger-ui/latest/swagger-ui-bundle.js"></script>
-            <script src="//pagecdn.io/lib/swagger-ui/latest/swagger-ui-standalone-preset.js"></script>
+            <script src="//cdnjs.cloudflare.com/ajax/libs/swagger-ui/__VERSION__/swagger-ui-bundle.js"></script>
+            <script
+                src="//cdnjs.cloudflare.com/ajax/libs/swagger-ui/__VERSION__/swagger-ui-standalone-preset.js"></script>
```
Comment (prryplatypus) [inline sanic_ext/extensions/openapi/ui/swagger.html:21]: Do we want to use minified versions for this too, maybe?

Comment (ahopkins) [reply sanic_ext/extensions/openapi/ui/swagger.html:21]: Well, when I looked I was certain there weren't any. But I looked again and now I see them :joy:

Comment (prryplatypus) [reply sanic_ext/extensions/openapi/ui/swagger.html:21]: ✨ _MAGIC_ ✨

**Review by prryplatypus — APPROVED** [review]

**Pull request merged**